notebooks/02_model/baseline_deep_dive.ipynb

```diff
@@ -195,7 +195,7 @@
     "    header=['UserId', 'MovieId', 'Rating', 'Timestamp']\n",
     ")\n",
     "\n",
-    "display(data.head())"
+    "data.head()"
```

Comment:
this is at least cross-platform, but it will render as just the first row in databricks.
you might want to do data.head(5) or something, unless we want to build a utility to check the platform and call display or not

Reply:
@gramhagen Does Databricks draw pd.dataframe with data.head() or should we call display(...) explicitly?

Reply:
no, you just see text details of the row, so actually display would look better (though you still want to define how much of the head to show)
```display(df.head(10))```

Reply:
Got it. Currently, other notebooks are not using display() so I will just follow them for now, but this is good to know.